### DISCUSSION TITLE
How to contribute to docs?

### DISCUSSION THREAD

**Discussion by sairajchouhan** [vercel/turborepo#679] [Help] [answered]
I saw a bug in truborepo docs, when I switch to light mode and refresh it still loads dark mode, I wanted to create an issue, (and eventually a PR) but all issue templates are for turbo cli and not for docs

So I was wondering can people contribute to docs?
if yes it would be really helpful if there are steps on how to run the docs locally, I tried, but all I see are errors (in windows) and I couldn't figure it out.

**Comment by mehulkar** [comment, marked as answer]
You can open a pull request if you're still having an issue! Also in the Issue Templates, choose Turborepo and you can fill in anything for the other fields if they're required.
